src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add a `prototype_options` option to `CollectionType`
+ * Add `form_label_text` and `form_help_text` block to form themes
```

Comment:
only files in the twig bridge are modified, so this should be moved there I guess?

Reply:
Good catch updated the correct changelog now!